src/chart/XYAxis.js

```diff
@@ -10,24 +10,25 @@
         SVGWidget.call(this);
         this._drawStartPos = "origin";
 
-        this._dateParser = d3.time.format("%Y-%m-%d").parse;
+        this._dateParserData = d3.time.format("%Y-%m-%d").parse;
+        this._dateParserValue = d3.time.format("%Y-%m-%d").parse;
```

Comment:
how come this is here and two of the same kind of object? do these return 0000-00-00 as default for a placeholder?